### ISSUE TITLE
throw error on init

### ISSUE THREAD

**Issue by DenisCarriere** [pinax-network/substreams-sink-clickhouse#80]
Assuming this is because AUTH_KEY is not being provided, it throws an error?

<img width="823" alt="image" src="https://github.com/pinax-network/substreams-sink-clickhouse/assets/550895/169de03e-636d-4c9d-b834-ace8ebf3aea9">

**.env**
```env
PUBLIC_KEY=${{shared.PUBLIC_KEY}}
VERBOSE=${{shared.VERBOSE}}
HOST=${{shared.HOST}}
DATABASE=default
USERNAME=default
PASSWORD=<PASSWORD>
```

Another logging error =>

> 2023-11-17 00:59:37.580 ERROR           substreams-sink-clickhouse      The socket connection was closed unexpectedly. For more information, pass `verbose: true` in the second argument to fetch()

**Comment by DenisCarriere** [comment]
Oh.. the issue was because the `HOST` url was using `http://` instead of `https://`

🤷  perhaps there's a way we can improve error messages on `/init`

**Comment by JulienR1** [comment]
I have replaced the try-catch block with controlled response objects. The logs are much better and separated for each step.

**Comment by DenisCarriere** [comment]
Perfect 👌

**Comment by JulienR1** [comment]
Closed by #85
